src/coreComponents/unitTests/meshTests/supportedElements.vtk

```diff
@@ -0,0 +1,571 @@
+9.46527 -10.796 -0.178719
+12.5178 -10.149 -1.58815
+12.1925 -9.2849 -1.50468
+11.5923 -8.75085 -1.04209
+10.9465 -8.75085 -0.377061
+10.5016 -9.2849 0.236379
+10.4276 -10.149 0.56392
+10.7529 -11.0131 0.480452
+11.3531 -11.5471 0.0178584
+11.999 -11.5472 -0.647167
+12.4438 -11.0131 -1.26061
+13.5724 -9.851 -0.56392
+13.2471 -8.98689 -0.480452
+12.6469 -8.45285 -0.0178584
+12.001 -8.45285 0.647167
+11.5562 -8.98689 1.26061
+11.4822 -9.851 1.58815
+11.8075 -10.7151 1.50468
+12.4077 -11.2491 1.04209
+13.0535 -11.2491 0.377061
+13.4984 -10.7151 -0.236379
+16.5178 -10.149 -1.58815
+16.2363 -9.35421 -1.52958
+15.7124 -8.81175 -1.14797
+15.1124 -8.69387 -0.564487
+14.6268 -9.03797 0.0356284
+14.4098 -9.73483 0.461839
+14.5302 -10.5632 0.578827
+14.9499 -11.26 0.349448
+15.5356 -11.6041 -0.15347
+16.1013 -11.4863 -0.770256
+16.4674 -10.9438 -1.30508
+17.5724 -9.851 -0.56392
+17.2909 -9.0562 -0.505352
+16.767 -8.51375 -0.123746
+16.167 -8.39586 0.459742
+15.6814 -8.73997 1.05986
+15.4643 -9.43682 1.48607
+15.5848 -10.2652 1.60306
+16.0045 -10.962 1.37368
+16.5902 -11.3061 0.870758
+17.1559 -11.1882 0.253973
+17.522 -10.6458 -0.280855
+
+CELLS 11 386
+4
+0
+1
+2
+3
+5
+4
+5
+6
+7
+8
+6
+9
+10
+11
+12
+13
+14
+8
+15
+16
+17
+18
+19
+20
+21
+22
+10
+23
+24
+25
+26
+27
+28
+29
+30
+31
+32
+12
+33
+34
+35
+36
+37
+38
+39
+40
+41
+42
+43
+44
+52
+9
+7
+45
+46
+47
+48
+49
+50
+51
+7
+52
+53
+54
+55
+56
+57
+58
+4
+45
+46
+53
+52
+4
+46
+47
+54
+53
+4
+47
+48
+55
+54
+4
+48
+49
+56
+55
+4
+49
+50
+57
+56
+4
+50
+51
+58
+57
+4
+51
+45
+52
+58
+59
+10
+8
+59
+60
+61
+62
+63
+64
+65
+66
+8
+67
+68
+69
+70
+71
+72
+73
+74
+4
+59
+60
+68
+67
+4
+60
+61
+69
+68
+4
+61
+62
+70
+69
+4
+62
+63
+71
+70
+4
+63
+64
+72
+71
+4
+64
+65
+73
+72
+4
+65
+66
+74
+73
+4
+66
+59
+67
+74
+66
+11
+9
+75
+76
+77
+78
+79
+80
+81
+82
+83
+9
+84
+85
+86
+87
+88
+89
+90
+91
+92
+4
+75
+76
+85
+84
+4
+76
+77
+86
+85
+4
+77
+78
+87
+86
+4
+78
+79
+88
+87
+4
+79
+80
+89
+88
+4
+80
+81
+90
+89
+4
+81
+82
+91
+90
+4
+82
+83
+92
+91
+4
+83
+75
+84
+92
+73
+12
+10
+93
+94
+95
+96
+97
+98
+99
+100
+101
+102
+10
+103
+104
+105
+106
+107
+108
+109
+110
+111
+112
+4
+93
+94
+104
+103
+4
+94
+95
+105
+104
+4
+95
+96
+106
+105
+4
+96
+97
+107
+106
+4
+97
+98
+108
+107
+4
+98
+99
+109
+108
+4
+99
+100
+110
+109
+4
+100
+101
+111
+110
+4
+101
+102
+112
+111
+4
+102
+93
+103
+112
+80
+13
+11
+113
+114
+115
+116
+117
+118
+119
+120
+121
+122
+123
+11
+124
+125
+126
+127
+128
+129
+130
+131
+132
+133
+134
+4
+113
+114
+125
+124
+4
+114
+115
+126
+125
+4
+115
+116
+127
+126
+4
+116
+117
+128
+127
+4
+117
+118
+129
+128
+4
+118
+119
+130
+129
+4
+119
+120
+131
+130
+4
+120
+121
+132
+131
+4
+121
+122
+133
+132
+4
+122
+123
+134
+133
+4
+123
+113
+124
+134
+
+CELL_TYPES  11
+10
+14
+13
+12
+15
+16
+42
+42
+42
+42
+42
+
+CELL_DATA 11
+SCALARS attribute int 1
+LOOKUP_TABLE default
+0
+1
+2
+3
+4
+5
+6
+7
+8
+9
+10
+SCALARS test_field float 1
+LOOKUP_TABLE default
+0
+1
+2
+3
+4
+5
+6
+7
+8
+9
+10
```

Comment:
```suggestion
```
Do we need to keep this scalar field? I have the impression `testVTKImport` does not check anything about it.

Reply:
> Could you please add a test for `VTK_VOXEL` (type 11) elements? Some structured vtk formats that we support since PR #2139 got merged need that. We already know everything works fine thanks to the `cube` test, but it would be nice that such capability is directly tested. I would not change the two existing tests, and simply add a new (trivial) one.

Need clarification. Do you mean addition VTK_VOXEL to elementType and
testing it within supportedElements?

Reply:
I think that adding a `VTK_VOXEL` in the `supportedElements.vtk` file would do the trick.

Reply:
I see two options:
1. adding a `VTK_VOXEL` to `supportedElements.vtk`  as @TotoGaz suggests. However, this requires adding a new polyhedron to `supportedElements42.vtk` to conform to the validation checks.
1. adding a complete new test containing only `VTK_VOXEL` cells (1 or 2).

Reply:
Also, for completeness do you think we should have checks for `VTK_TRIANGLE` and `VTK_QUAD`? They can be used in unstructured grid to mark boundaries on which boundary conditions are prescribed.

Reply:
I would leave this aside for the moment. Things will probably evolve (in which direction, I do not know) with the fractures. I would stick to 3d simulation elements for now?

Reply:
OK, then option 1 seems the easiest path to wrap up the PR and get it merger.

Reply:
Done! option 1.